packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -81,8 +81,13 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
     <div>
       {!published && (
         <Toast accent="warning">
-          This draft is available to members in the working group listed below.
-          Only PMs can publish this output.
+          This draft is available to members in the{' '}
+          {props.workingGroups
+            ? 'working group'
+            : props.teams.length > 1
+            ? 'teams'
+            : 'team'}{' '}
```

Comment:
Can you please inject this from the parent. Also `team${props.teams.length > 1 ? 's':''}`